extensions/wikia/AssetsManager/config.php

```diff
@@ -3190,3 +3190,20 @@
 		'//extensions/wikia/RWEPageHeader/scripts/searchSuggestions.js'
 	]
 ];
+
+$config[ 'article_video_scss' ] = [
+	'type' => AssetsManager::TYPE_SCSS,
+	'skin' => [ 'oasis' ],
+	'assets' => [
+		'//extensions/wikia/ArticleVideo/styles/article-video.scss'
+	]
+];
+
+$config[ 'article_video_js' ] = [
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => [
+		'//extensions/wikia/ArticleVideo/scripts/article-video.js',
+		'//extensions/wikia/ArticleVideo/scripts/ooyalaVideo.js'
```

Comment:
Does the order make any difference? If yes, then ooyalaVideo should be above article-video. How about unifying the naming? Dash or camel case.